### ISSUE TITLE
useStorage: window is not defined

### ISSUE THREAD

**Issue by laurensnl** [beautifulinteractions/beautiful-react-hooks#94]
**Describe the bug**
Since beautiful-react-hooks v0.26.0, NextJS throws this error while building:

```
error - ReferenceError: window is not defined
    at Object.useStorage [as default] .../node_modules/beautiful-react-hooks/dist/useStorage.js:15:17)
    at Object.<anonymous> (.../node_modules/beautiful-react-hooks/dist/useLocalStorage.js:8:41)
```

The build fails.

**To Reproduce**
1. Install beautiful-react-hooks@0.27.0 (or 0.26.)0
2. Build using next@canary.29

**Expected behavior**
Expect to build without errors.

**Comment by antonioru** [comment]
thanks @laurensnl I'll look into it

**Comment by laurensnl** [comment]
> thanks @laurensnl I'll look into it 

Thanks! Let me know if you need more information.

**Comment by antonioru** [comment]
@laurensnl thanks, the truth is: I think I know what I did wrong :) 
I introduced this bug few days ago, sorry about that... I'll check as soon as possibile

**Comment by antonioru** [comment]
@laurensnl I think I fixed it, please install 0.27.1 

Please let me know if you find any other issue

**Comment by laurensnl** [comment]
Thanks for the quick fix @antonioru! Works like a charm!
